components/textFields/AdvancedOpenTextField.js

```diff
@@ -195,13 +210,20 @@ const AdvancedOpenTextField = (props) => {
     }
 
     const ErrorMsg = () => {
-        if(isInvalid) {
+        if(invalidLength) {
             return(
                 <Text>
                     Too long!
                 </Text>
             )
         }
+        if(invalidVin) {
+          return(
+            <Text>
+              Warning: Invalid VIN
```

Comment:
I think this text might need to a different color like red as currently it is not very distinctive. A user could potentially saw the bar green turned green after entering and submitting some value to the text field and miss this text warning thinking the VIN was fine.

Reply:
Changed to red, but I wonder if it should stop the bar from turning green? There is a chance they have an old VIN (pre 1980 i think) that gets the warning but should still work

Reply:
sounds good! Yeah we prob don't worry about stopping the bar from turning green now since if the bar doesn't turn green, whatever they entered even if they intend it that way wouldn't be saved by the app redux state